src/Kafka/types.ts

```diff
@@ -8,7 +8,7 @@ export type DateIsoString = string;
  * Typescript errors to find where extra information need to be added (images,
  * translations).
  */
-export type CloudProvider = "aws" | "azure";
+export type CloudProvider = "aws" | "gcp" | "azure";
```

Comment:
@riccardo-forina adding GCP to the cloud provider type will also affect the MarketPlace billing, MarketPlace type includes `cloudProvider` type, so should we also make changes to the billing MarketPlace billing?

Reply:
yes it's ok for GCP to be in the marketplace as well, it will not be displayed unless the API says so